bundle/src/main/java/com/adobe/acs/commons/hc/impl/HealthCheckStatusEmailer.java

```diff
@@ -0,0 +1,246 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2017 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.hc.impl;
+
+import com.adobe.acs.commons.email.EmailService;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.felix.scr.annotations.Activate;
+import org.apache.felix.scr.annotations.Component;
+import org.apache.felix.scr.annotations.ConfigurationPolicy;
+import org.apache.felix.scr.annotations.Properties;
+import org.apache.felix.scr.annotations.Property;
+import org.apache.felix.scr.annotations.Reference;
+import org.apache.felix.scr.annotations.Service;
+import org.apache.sling.commons.osgi.PropertiesUtil;
+import org.apache.sling.hc.api.execution.HealthCheckExecutionOptions;
+import org.apache.sling.hc.api.execution.HealthCheckExecutionResult;
+import org.apache.sling.hc.api.execution.HealthCheckExecutor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component(
+        label = "ACS AEM Commons - Health Check Status E-mailer",
+        description = "Scheduled Service that runs specified Health Checks and e-mails the results",
+        configurationFactory = true,
+        policy = ConfigurationPolicy.REQUIRE,
+        metatype = true
+)
+@Properties({
+        @Property(
+                label = "Cron expression defining when this Scheduled Service will run",
+                name = "scheduler.expression",
+                description = "Every weekday @ 8am = [ 0 0 8 ? * MON-FRI * ] Visit www.cronmaker.com to generate cron expressions.",
+                value = "0 0 8 ? * MON-FRI *"
+        ),
+        @Property(
+                name = "scheduler.concurrent",
+                boolValue = false,
```

Comment:
Should this be false if it's a configuration factory? If something it set to run every 10 mins and something else is set to run every 24 hours, there could be a concurrency conflict; will this setting result in one of them not firing?